web/developer.actyx.com/docs/how-to/app-auth/sign-app-manifest.mdx

```diff
@@ -6,6 +6,22 @@ hide_table_of_contents: false
 image: /images/defaults/default.svg
 ---
 
+:::tip
```

Comment:
This repeating tips may benefits from being made into MDX's reusable components

Reply:
I don't think it's worth the extra complexity and effort for 4 blocks of text.

Reply:
Its worth can only be found out when in the future this text needs to be changed.

Reply:
Let's keep it like this for now then, we'll reconsider later